app/interactors/send_team_hours_update.rb

```diff
@@ -0,0 +1,15 @@
+class SendTeamHoursUpdate
+  include Interactor
+  include HasHarvest
+
+  def call
+    Team.active.each do |team|
+      from = Date.this_week.first
+      to = Date.this_week.last
```

Comment:
I initially had that but realized that this is a team-level thing, and everyone should know they are on the team already. Better to just send everyone the one email vs doing a ton of work for every individual user. Time zones shouldn't be an issue here as we're talking about a full week vs individual days or hours, particularly work-day hours and it should be extremely rare that work hits the day boundary.